tests/filecheck/mlir-conversion/with-mlir/dialects/pdl_interp/pdl_interp_ops.mlir

```diff
@@ -0,0 +1,119 @@
+// RUN: mlir-opt %s --mlir-print-op-generic | xdsl-opt --print-op-generic | filecheck %s
```

Comment:
I would add a second test for custom syntax, and I would recommend testing parsing and printing both ways.
```suggestion
// RUN: xdsl-opt %s --mlir-print-op-generic | mlir-opt --mlir-print-op-generic | xdsl-opt | filecheck %s
// RUN: xdsl-opt %s | mlir-opt | xdsl-opt | filecheck %s
```

Reply:
That custom syntax test won't work on the same input as `pdl_interp.func` and others don't have the custom syntax yet so the output from mlir-opt can't be parsed.
Is it ok to add this in the follow up pr where custom parsing is added for those?

Reply:
Ah yes of course, maybe add it as a comment in this PR, and add it in the next one?